strapi/strapi
# [DEITS] Export Local Assets

### What does it do?

Export assets located in the `public/uploads` app directory

### How to test it?

- Go to an application directory such as `examples/getstarted`
- Run `yarn strapi export --encrypt=false`
- You should see the `assets` folder populated with your files in the created archive

### TODO

- Add unit tests
